src/utils/transition-manager.js

```diff
@@ -132,15 +134,19 @@ export default class TransitionManager {
       isRotating: startProps.bearing !== endProps.bearing ||
         startProps.pitch !== endProps.pitch
     };
-
+    const currentTime = Date.now();
+    const completion = this.state.interruption === TRANSITION_EVENTS.UPDATE ?
+                      (currentTime - this.state.startTime) / startProps.transitionDuration : 0;
+    const totalCompletion = this.state.totalCompletion + completion;
```

Comment:
What happens if `this.state.totalCompletion` is non-zero and you interrupt it with BREAK mode?